csun-metalab/media
# ljg-10-19-18-fix-cors

# Summary
This changes the way the resources are retrieved. Before it was a redirect to the resource and now it returns the URL of the resource or an error message where applicable.

# How to test
Make sure you run a `composer install` and get the latest `env` values, then simply click on the URLS on the landing page `/` and receive the respective JSON
